wiki/BanchoBot/en.md

```diff
@@ -1,154 +1,148 @@
+13:00 pippi: !help
+13:00 BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+13:00 BanchoBot: WHERE <user>
+13:00 BanchoBot: STATS <user>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <reason> - call for an admin
+13:00 BanchoBot: REQUEST [list] - shows a random recent mod request
+13:00 BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
 ```
 
-### ROLL
+### Roll
 
 ```
-!ROLL <number>
+!roll <argument> <number>
 ```
 
-Draws a random number from 1 to the selected number (default is 100).
-
-#### Usage Example
+`!roll` draws a random number from 1 to the selected number. If a number is not specified or an argument is given, then the maximum number will be set to 100. Examples of sending this command are shown below:
 
 ```
-pippi: !roll 1000
-BanchoBot: pippi rolls 109 point(s)
+13:00 pippi: !roll 1000
+13:00 BanchoBot: pippi rolls 109 point(s)
 ```
 
-### STATS
-
 ```
-!STATS <user>
+13:01 pippi: !roll probability of failure 10
+13:01 BanchoBot: pippi rolls 75 point(s)
 ```
 
-Shows the entered user's stats. The resulting stats depend on the mode they are playing or last played.
-
-*Note: BanchoBot will not tell you which game mode the stats came from.*
-
-*Notice: BanchoBot will respond with `User not found` if the user has never played a game, even if the user does exist.*
-
-#### Usage Examples
+### User statistics
 
 ```
-pippi: !stats peppy
-BanchoBot: Stats for peppy:
-BanchoBot: Store: 422,187,979 (#94718)
-BanchoBot: Plays: 7149 (lv65)
-BanchoBot: Accuracy: 87.29%
+!stats <user>
 ```
 
-```
-pippi: !stats Ephemeral:
-BanchoBot: Stats for Ephemeral is Playing:
-BanchoBot: Store: 2,384,155,536 (#33697)
-BanchoBot: Plays: 14054 (lv96)
-BanchoBot: Accuracy: 94.19%
-```
+`!stats` displays the entered user's game statistics (stats). The results of which depend on the [game mode](/wiki/Game_Modes) that the entered user is playing (or last played), although BanchoBot will not display which game mode the stats came from. An example of sending this command is shown below:
 
-### WHERE
+*Note: BanchoBot will respond with `User not found` if the user has never played a game, even if the user does exist.*
 
 ```
-!WHERE <user>
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy:
+13:01 BanchoBot: Store: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
 ```
 
-Shows the entered user's current location.
+### Where
 
-*Note: By default, this only shows the user's country. If the user has `Share your city location with others` enabled, it may also display their city.*
+```
+!where <user>
+```
 
-#### Usage Example
+`!where` shows the entered user's current location. By default, this only shows the user's country. But if the user has `Share your city location with others` enabled, it will also display their city.* An example of sending this command is shown below:
```

Comment:
```suggestion
`!where` shows the entered user's current location. By default, this only shows the user's country, but if the user has `Share your city location with others` game setting enabled, it will also display their city.* An example of sending this command is shown below:
```

Reply:
Coordinating conjunctions are able to begin sentence. In this case, I think it prevents the sentence from becoming overly long.

Reply:
merging these two sentences into one would make it less of a bumpy read, but oh well